mantidproject/mantid
# Elemental analysis exposed to workbench

**Description of work.**

Porting the elemental analysis GUI to workbench 
-->

**To test:**


you need to set up the manage user dir to include the sundials folder from the data I sent you

then if you enter (I think) 2695 and press load it should do something
Open the elemental analysis GUI, load some data and have a play
Make sure to give the plotting a good run around. 

Fixes #25941. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
